shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -490,6 +503,59 @@ - (void)dealloc {
   [super dealloc];
 }
 
+#pragma mark - Status Bar touch event related swizzling
+
+static constexpr CGFloat kStandardStatusBarHeight = 20.0;
+
+static IMP __original_sendevent = nil;
+
+void _flutter_sendevent(id self, SEL _cmd, UIEvent *event) {
+  // Call the original implementation.
+  ((void(*)(id,SEL,UIEvent*))__original_sendevent)(self, _cmd, event);
+
+  // Intercept touchBegan events.
+  UITouch *touch = event.allTouches.anyObject;
```

Comment:
Now that I think about it, we need to handle taps and not just beginnings of touches. That is, begin the scroll when the finger lifts up from the status bar without moving too much from its start location.